src/tests/ftest/util/server_utils.py

```diff
@@ -194,7 +194,8 @@ def __init__(self):
                  "CRT_CTX_SHARE_ADDR=0",
                  "CRT_TIMEOUT=30",
                  "FI_SOCKETS_MAX_CONN_RETRY=1",
-                 "FI_SOCKETS_CONN_TIMEOUT=2000"]
+                 "FI_SOCKETS_CONN_TIMEOUT=2000",
+                 "DD_SUBSYS=mgmt,io,md,epc,rebuild"]
```

Comment:
apologies, these bits are DD_MASK bits.   

If you take a look at daos/src/include/daos/debug.h, you'll see two macros defining FAC and DB.   
Additionaly, cart also provides a set of DB bits in src/include/gurt/debug_setup.h and FAC's in src/include/gurt/debug.h

DAOS uses cart ones as well.   When you see something like 

D_DEBUG(DB_TRACE, ....) this refers to the trace debug bit and is enabled if DD_MASK includes trace.

The FAC is at the top of each source file
 29 #define D_LOGFAC        DD_FAC(vos)

This says that all debug messages (regardless of bit) are using the VOS facility.